Gemfile

```diff
@@ -0,0 +1,71 @@
+source 'https://rubygems.org'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '4.2.5.1'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3'
+# Use SCSS for stylesheets
+gem 'sass-rails', '~> 5.0'
+# Use Uglifier as compressor for JavaScript assets
+gem 'uglifier', '>= 1.3.0'
+
+# See https://github.com/rails/execjs#readme for more supported runtimes
+# gem 'therubyracer', platforms: :ruby
+
+# Use jquery as the JavaScript library
+gem 'jquery-rails', '~> 4.0.5'
+
+# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+gem 'jbuilder', '~> 2.0'
+# bundle exec rake doc:rails generates the API under doc/api.
+gem 'sdoc', '~> 0.4.0', group: :doc
+
+# Use ActiveModel has_secure_password
+# gem 'bcrypt', '~> 3.1.7'
+
+# Application server: Puma
+# Puma was chosen because it handles load of 40+ concurrent users better than Unicorn and Passenger
+# Discussion: https://github.com/18F/college-choice/issues/597#issuecomment-139034834
+gem "puma", "~> 2.16.0"
+
+#TODO AYL add version number
+gem 'pg', '~> 0.18.4'
```

Comment:
Also, does the user need to do anything extra to get PostGIS setup in the database? Not sure if the newer versions come with PostGIS already setup or if that extension needs to be setup.